### DISCUSSION TITLE
关于翻译框选取文字的问题。【操作相关】

### DISCUSSION THREAD

**Discussion by somaicro** [STranslate/STranslate#207] [General]
你好，如图所示，想只选取翻译个别文字时，翻译框的拖拉框的识别范围有点大，会妨碍上行的文字选择。能否缩小范围，避免影响选取的呢。感谢~
![PixPin_2024-09-28_15-17-47](https://github.com/user-attachments/assets/0c596857-3356-4f28-b5b0-1f4c7165f14a)

**Comment by ZGGSONG** [comment]
调不好 主界面拖拽下个版本移除掉吧  以后调整就到设置-翻译服务里去调整吧
